core/storage/improvements/gae_models_test.py

```diff
@@ -33,6 +33,45 @@
 class TaskEntryModelTests(test_utils.GenericTestBase):
     """Unit tests for TaskEntryModel instances."""
 
+    def test_get_field_name_mapping_to_takeout_keys(self) -> None:
+        self.assertEqual(
+            improvements_models.TaskEntryModel.
+            get_field_name_mapping_to_takeout_keys(),
+            {
+                'resolver_id': 'task_ids_resolved_by_user',
+                'issue_description': 'issue_descriptions',
+                'status': 'statuses',
+                'resolved_on': 'resolution_msecs'
+            })
+
+    def test_get_export_policy(self) -> None:
+        sample_dict = {
```

Comment:
Same comments re naming as in other PRs.

Reply:
Done!